opentracing-cdi-example/src/main/java/io/opentracing/contrib/cdi/example/InventoryNotificationService.java

```diff
@@ -0,0 +1,33 @@
+package io.opentracing.contrib.cdi.example;
+
+import io.opentracing.ActiveSpan;
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+/**
+ * This is the final call in the chain. This is an asynchronous stateless EJB, which obtains the span context
+ * via a method parameter. This bean is not intercepted in any way by us, so, the span context received is exactly
+ * the same as what was sent by the caller.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+public class InventoryNotificationService {
+    private static final Logger log = Logger.getLogger(InventoryNotificationService.class.getName());
+
+    @Inject
+    Tracer tracer;
+
+    @Inject
+    ActiveSpan parentSpan;
```

Comment:
Why `parentSpan` and not `activeSpan` ?
It may be that you're using it for the `child-of` relation below, but maybe add a comment on this injection?